pages/process.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Process
+permalink: /process/
+layout: primary
+lead: Websites, applications, and strategies that help agencies provide excellent value to the public.
+content_wide: true
+content_focus: false
+gridless: true
+---
+
+<section class="nz-section background-gray">
+  {% include tools-process-nav.html %}
+
+  <div class="nz-grid">
+    <h2 tabindex="0" style="color: #2337CE">Process</h2>
```

Comment:
The inline style isn't ideal here but we probably don't need to optimise for maintainability here 😄